samples/highcharts/studies/dashboard-add-layout/demo.mjs

```diff
@@ -0,0 +1,165 @@
+
+// Bring in other forms of Highcharts
```

Comment:
This whole demo should not be treated as a full-fledged demo thus I would create a separate directory similar to `highcharts/cypress` with dashboard pages for cypress.